experimentA/idr0044-experimentA-filePaths.tsv

```diff
@@ -1,4 +1,4 @@
-Dataset1	/uod/idr/filesets/idr0044-mcdole/20180412-ftp/Dataset1/ImageData/
-140813	/uod/idr/filesets/idr0044-mcdole/20180926-ftp/140813/
-151205	/uod/idr/filesets/idr0044-mcdole/20180926-ftp/151205/
-160723	/uod/idr/filesets/idr0044-mcdole/20180926-ftp/160723/
+Embryo1	/uod/idr/filesets/idr0044-mcdole/20180412-ftp/Dataset1/ImageData/
```

Comment:
Following the renaming `idr0044-mcdole` will need to be renamed as `idr0044-mcdole-tardislightsheet`